backend/protocol_rpc/endpoints.py

```diff
@@ -655,3 +683,7 @@ def register_all_rpc_endpoints(
         partial(set_finality_window_time, consensus),
         method_name="sim_setFinalityWindowTime",
     )
+    register_rpc_endpoint(
+        partial(get_contract, consensus_service),
+        method_name="eth_getContract",
```

Comment:
@mpaya5 this is not an Ethereum endpoint nor a GenLayer one. It is only a simulator's endpoint, so we should name it `sim_getContract`

Reply:
Done!